### ISSUE TITLE
New window never go on master

### ISSUE THREAD

**Issue by Cyberdr3am** [phenax/bsp-layout#10]
Hi

Thanks for making this app.
I would like to know if it would be possible to make the new windows take the place of the master automatically.
Currently the window i use remain in master while the new is always on the right (with the tall layout)

**Comment by phenax** [comment]
This behaviour was implemented intentionally for my personal workfow. I'll try to implement a configuring spawning behavior.

**Comment by Cyberdr3am** [comment]
Ok thanks a lot.

**Comment by Cyberdr3am** [comment]
If it's too complicated can you tell me how to do it definitively.
I tried to do it by myself but I couldn't do it because I don't know bspwm well

**Comment by phenax** [comment]
It is a bit tricky. I've got it to replace master if the node was spawned when focusing on master but for other nodes it is a bit more complicated. I'll try to spend some time this weekend with it and I'll let you know.

**Comment by Cyberdr3am** [comment]
Ok thanks

**Comment by indeedwatson** [comment]
You can do this with the default layout with `bspc config initial_polarity first_child`, I have been using that setting forever.

Unfortunately, combining that option with this script, all it does is makes it so the new window spawns at the top slave instead of the bottom one.

**Comment by a2n-s** [comment]
Hi @Cyberdr3am :wave: 
`bsp-layout` is back :muscle: 

if you've still got interest in that feature and/or news from your side, please let me know and we'll see what can be done :wink:

**Comment by Cyberdr3am** [comment]
Hi @amtoine 

Yes i am still interest in that feature. If you can do it. I whant to use it with Master-stack rtall.
Thx.

**Comment by TAforever** [comment]
Yes guys, I'm also interested in this feature. It seems to me that if the new window does not become the master, then in many ways the meaning of the master stack layout is lost. It would be great if you could implement it.
